### PR TITLE
Add FAQ page

### LINKED ISSUE #91: Add markdown FAQ page

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Fixes #91 

This adds the `faq.md` file, so we just need to add the changes there. Linking to it not yet done.

### CODE REVIEW

File: client/public/faq.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+For now, you'll have to change it back using the slider or text box, or reload the page to reset all policy parameters. We're working on a [better way](https://github.com/UBICenter/policyengine-uk/issues/23).
+
+
+## UK impact page
+
+
+### What data do you use to estimate UK-wide impacts?
+
+We use the Family Resources Survey (FRS), the UK's standard survey for estimating poverty. The latest FRS is from 2018-19, and we extrapolate this to 2021 using growth factors published by x. We also adjust benefit receipt to reflect trends like the Universal Credit rollout.
```
Comment (MaxGhenis) [inline client/public/faq.md]: todo

File: client/public/faq.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+Using and sharing PolicyEngine is already a great help to us! If you'd like to support our work and web hosting fees, please consider [making a donation](https://www.ubicenter.org/donate/). We're also entirely open source, and welcome contributions from developers on [GitHub](github.com/ubicenter/policyengine-uk).
+
+
+## Policy page
+
+
+### Do you only simulate policies listed on this page?
+
+
```
Comment (MaxGhenis) [inline client/public/faq.md]: remove blank lines between question and answer

File: client/public/faq.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+### Will you be adding more policy parameters?
+
+Yes, we’re planning to add more tax and benefit parameter options, and other taxes like [land value taxes](https://github.com/UBICenter/policyengine-uk/issues/105) and [carbon taxes](https://github.com/UBICenter/policyengine-uk/issues/104). What would you like to simulate? 
+
+
+### How can I reset a policy parameter to its current value?
+
+For now, you'll have to change it back using the slider or text box, or reload the page to reset all policy parameters. We're working on a [better way](https://github.com/UBICenter/policyengine-uk/issues/23).
+
+
```
Comment (MaxGhenis) [inline client/public/faq.md:48]:
```suggestion
### As of what date are policy parameters set?

We simulate policy as of 2021-01-01, and assume that policy is active throughout 2021. We're working on [updating to the latest policy](https://github.com/UBICenter/policyengine-uk/issues/44).
```

File: client/public/faq.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+### Who created PolicyEngine?
+
+PolicyEngine is a product from the [UBI Center](http://ubicenter.org), a research organization studying universal basic income policies. We also developed [OpenFisca-UK](https://github.com/PSLmodels/openfisca-uk), the microsimulation model underlying PolicyEngine. [Nikhil Woodruff](https://www.ubicenter.org/author/nikhil/) is the technical lead and [Max Ghenis](https://www.ubicenter.org/author/max/) is the product manager.
+
+
+### When will PolicyEngine be available in my country?
+
+Currently, PolicyEngine is only available in the UK, but we're working on bringing it to the US. Want to see PolicyEngine in your country? Let us know!
```
Comment (MaxGhenis) [inline client/public/faq.md]: link typeform

File: client/public/faq.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+
+### What behavioral or macroeconomic assumptions do you make?
+
+None; PolicyEngine is a "static model" only. For example, it assumes that changing marginal tax rates will not affect labour supply.
+
+
+### How are the reform provisions sequenced for the budgetary impact chart?
+
+Since policy reforms can interact, we do not model them independently. Instead, we start with programs that produce budgetary costs (like new spending programs or tax cuts), ordered from largest to smallest, then do the same with programs that produce budgetary surpluses. This sequence does not follow the order the reforms were specified.
```
Comment (MaxGhenis) [inline client/public/faq.md:70]: copying my question from the doc (more for my curiosity though could be worth documenting):
>as I was typing this I realized...this is actually really nontrivial right? The relative size of reform provisions could depend on how they interact and are sequenced. How do you do this without getting into a loop?

**Review by MaxGhenis — APPROVED** [review]
Good to merge after these

File: client/public/faq.md (hunk 5 above)
Comment (nikhilwoodruff) [reply client/public/faq.md:70]: Yes, and to be clear we are simulating, then re-ordering in the waterfall charts. Pretty sure this is not actually possible in all cases (to be able to enforce an ordering when simulating).

File: client/public/faq.md (hunk 4 above)
Comment (nikhilwoodruff) [reply client/public/faq.md]: Linked

File: client/public/faq.md (hunk 3 above)
Comment (nikhilwoodruff) [reply client/public/faq.md:48]: Added

File: client/public/faq.md (hunk 2 above)
Comment (nikhilwoodruff) [reply client/public/faq.md]: Done

File: client/public/faq.md (hunk 1 above)
Comment (nikhilwoodruff) [reply client/public/faq.md]: Added ONS and OBR

**Pull request merged**